packages/components/src/components/gradient-background/gradient-background.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { CSSProperties } from 'react';
+import './gradient-background.scss';
+
+type TProps = {
+    color: CSSProperties['backgroundColor'];
+    primary: CSSProperties['background'];
+    secondary: CSSProperties['background'];
+    tertiary: CSSProperties['background'];
+};
+
+const GradientBackground: React.FC<React.PropsWithChildren<TProps>> = ({
+    children,
+    color,
+    primary,
+    secondary,
+    tertiary,
+}) => (
+    <div className='gradient-background'>
+        <div className='gradient-background__container' style={{ backgroundColor: color }}>
+            <div className='gradient-background__tertiary' style={{ background: tertiary }} />
+            <div className='gradient-background__secondary' style={{ background: secondary }} />
+            <div className='gradient-background__primary' style={{ background: primary }} />
```

Comment:
```suggestion
            <div className='gradient-background__primary' style={{ background: primary }} />
            <div className='gradient-background__secondary' style={{ background: secondary }} />
            <div className='gradient-background__tertiary' style={{ background: tertiary }} />
```
@farzin-deriv we need to change the order of the elements because with current implementation our secondary color is under primary and there is mismatch in colors.